packages/linux/package.mk

```diff
@@ -275,12 +275,18 @@ makeinstall_target() {
 
     # install platform dtbs, but remove upstream kernel dtbs (i.e. without downstream
     # drivers and decent USB support) as these are not required by LibreELEC
-    cp -p arch/${TARGET_KERNEL_ARCH}/boot/dts/*.dtb ${INSTALL}/usr/share/bootloader
+    for dtb in arch/${TARGET_KERNEL_ARCH}/boot/dts/*.dtb arch/${TARGET_KERNEL_ARCH}/boot/dts/*/*.dtb; do
+      if [ -f ${dtb} ]; then
+        cp -v ${dtb} ${INSTALL}/usr/share/bootloader
+      fi
+    done
     rm -f ${INSTALL}/usr/share/bootloader/bcm283*.dtb
+    # duplicated in overlays below
+    safe_remove ${INSTALL}/usr/share/bootloader/overlay_map.dtb
```

Comment:
this seems to be something specific to RPi thus it will throw a warning on all other platforms. We may want to look into creating a custom script that can be overridden per platform similar to what we do with u-boot.

Reply:
this is in the `elif [ "${BOOTLOADER}" = "bcm2835-bootloader" ]; then` block so it's fine to use safe_remove